challenges/08-coding-interview-prep/data-structures.json

```diff
@@ -1163,42 +1272,56 @@
         "The next few challenges will cover maps and hash tables. Maps are data structures that store key-value pairs. In JavaScript, these are available to us as objects. Maps provide rapid lookup of stored items based on key values and are very common and useful data structures.",
         "Instructions: Let's get some practice creating our own map. Because JavaScript objects provide a much more efficient map structure than anything we could write here, this is intended primarily as a learning exercise. However, JavaScript objects only provide us with certain operations. What if we wanted to define custom operations?",
         "Use the <code>Map</code> object provided here as a wrapper around a JavaScript <code>object</code>. Create the following methods and operations on the Map object:",
-        "<code>add</code> accepts a <code>key, value</code> pair to add to the map",
-        "<code>remove</code> accepts a key and removes the associated <code>key, value</code> pair",
-        "<code>get</code> accepts a <code>key</code> and returns the stored <code>value</code>",
-        "<code>has</code> returns a <code>boolean</code> for the presence or absence of an item",
-        "<code>values</code> returns an array of all the values in the map",
-        "<code>size</code> returns the number of items in the map",
-        "<code>clear</code> empties the map"
+        "<ul>",
+        "<li><code>add</code> accepts a <code>key, value</code> pair to add to the map.</li>",
+        "<li><code>remove</code> accepts a key and removes the associated <code>key, value</code> pair</li>",
+        "<li><code>get</code> accepts a <code>key</code> and returns the stored <code>value</code></li>",
+        "<li><code>has</code> accepts a <code>key</code> and returns <dfn>true</dfn> or <dfn>false</dfn> based on the presence of a key.</li>",
```

Comment:
`"<li><code>has</code> accepts a <code>key</code> and returns <dfn>true</dfn> if an element with the key exists or <dfn>false</dfn> if one doesn't exist.</li>",`

Reply:
Or maybe an even shorter version:

``"<li><code>has</code> accepts a <code>key</code> and returns <dfn>true</dfn> if the key exists or <dfn>false</dfn> if it doesn't.</li>",``

Reply:
I think the second one is better.